.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,48 @@
+name: Close stale issues
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "0 6 * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v3
+      with:
+        issue-types: issues
+      
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: This issue has not received any attention in 1 year. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+        stale-issue-message: This issue has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
```

Comment:
We should use the real value here instead of "in a while", just as above for "1 year".
Also, is it possible to put each sentence on its own line? I could make it (slightly) easier to read, what do you think?

Reply:
Make senses to me, updated.